locale/en/biofluid.cfg

```diff
@@ -7,4 +7,12 @@ no-biofluid-network=Out of biofluid network
 
 [gui]
 priority=Priority
-temperature-filter=Temperature filter
\ No newline at end of file
+temperature-filter=Temperature filter
+
+[mod-setting-name]
+pal_min_fluid_reserve=Biofluid provider minimum fluid reserve
+pal_min_fluid_request=Biofluid requester minimum fluid request
```

Comment:
I recommend using `py-` as the mod prefix instead of `pal_` in order to match existing prototype internal names.

Reply:
I took the coal processing mod settings as the basis and since they seemed to lack any consistent naming convention I made one up.
Pushed a new commit to change the names to the suggested format.